packages/data/src/persist.js

```diff
@@ -8,6 +8,18 @@ import deprecated from '@wordpress/deprecated';
  */
 import { get } from 'lodash';
 
+// Defaults to the local storage.
```

Comment:
So we default to a broken (error-throwing) state for Node environments where this package is used?

It's probably a larger dependency than we'd want, but something like [Store.js](https://github.com/marcuswestin/store.js) is a good abstraction for this sort of thing.

Reply:
Not really, you can make it work by using setter method introduced in this PR:
```js
import store from 'store';

setPersistenceStorage( store );
```

It isn't ideal, needs to be documented but should allow us to move forward. I will add the following example in the README file.

Reply:
Opened #7015 to solve it.